sip-test-kit/src/main/java/de/ikor/sip/foundation/testkit/workflow/whenphase/routeproducer/RouteProducer.java

```diff
@@ -0,0 +1,17 @@
+package de.ikor.sip.foundation.testkit.workflow.whenphase.routeproducer;
+
+import org.apache.camel.Endpoint;
+import org.apache.camel.Exchange;
+
+/** Route producer interface */
+public interface RouteProducer {
```

Comment:
I feel like naming in this interface is misleading. 
In context of a test we're executing, this could be executor, runner, invoker or something similar. In case of cxf, there is no route involved at all.